impeller/entity/shaders/blending/vertices_uber.frag

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <impeller/blending.glsl>
+#include <impeller/types.glsl>
+#include "blend_select.glsl"
+
+uniform FragInfo {
+  float16_t alpha;
+  float16_t blend_mode;
+}
+frag_info;
+
+uniform f16sampler2D texture_sampler;
+
+in highp vec2 v_texture_coords;
+in mediump f16vec4 v_color;
+
+out f16vec4 frag_color;
+
+// A shader that implements the required src/dst blending for drawVertices and
+// drawAtlas advanced blends without requiring an offscreen render pass. This is
+// done in a single shader to reduce the permutations of PSO needed at runtime
+// for rarely used features.
+void main() {
+  f16vec4 dst = IPHalfUnpremultiply(v_color);
+  f16vec4 src = IPHalfUnpremultiply(texture(texture_sampler, v_texture_coords));
+  f16vec3 blend_result =
```

Comment:
If this is fast enough maybe we'll just do all the advanced blends this way 😏

Reply:
Oh, I don't see the T880 in the Vulkan malioc results. Perhaps that is too old for Vulkan? Maybe we should find an older baseline than the Mali-G78.

Reply:
We shrunk down the malioc to two models, a pixel and I think the moto g4 gpu

Reply:
Hmm, if memory serves, the reasoning was that one would capture the high end on Vulkan and the other the low one with GLES. But that was when we were still focusing on GLES. Should we have two for Vulkan (and decide what the low one should be)?

In any case, don't want to distract you. This is good. There are no spills on our current Vulkan baseline.

Reply:
That's a good idea , we should revisit the gpu list